src/models/costcode.js

```diff
@@ -1,7 +1,8 @@
 const Joi = require('joi');
 
 const costCodeSchema = Joi.object().keys({
-    code: Joi.number().integer().required(),
+    code: Joi.string().required(),
```

Comment:
Can you put some numerical constraints on this param? I know cost codes are very specific

Reply:
Yup add max length of 8. Most cost-codes are length 6, so this gives some flexibility for the future.